### PR TITLE
Add back LaunchOptions.GetInstance with old signature

### PR DESCRIPTION
**Pull request by faxue-msft**
Otherwise we'll need to ship a new kofe for update 3

### CODE REVIEW

File: src/MICore/LaunchOptions.cs [hunk 1]
```diff
@@ -597,7 +597,12 @@ public LaunchCompleteCommand LaunchCompleteCommand
             }
         }
 
-        public static LaunchOptions GetInstance(HostConfigurationStore configStore, string exePath, string args, string dir, string options, IDeviceAppLauncherEventCallback eventCallback, TargetEngine targetEngine, Logger logger)
+        public static LaunchOptions GetInstance(HostConfigurationStore configStore, string exePath, string args, string dir, string options, IDeviceAppLauncherEventCallback eventCallback, TargetEngine targetEngine)
+        {
+            return GetInstance(configStore, exePath, args, dir, options, eventCallback, targetEngine, null);
+        }
+    
+        public static LaunchOptions GetInstance(HostConfigurationStore configStore, string exePath, string args, string dir, string options, IDeviceAppLauncherEventCallback eventCallback, TargetEngine targetEngine, Logger logger = null)
```
Comment (edumunoz) [inline src/MICore/LaunchOptions.cs]: Do we want to still provide a default value for `logger`, given that you're adding another method without that parameter?

**Comment by edumunoz** [comment]
:rocket:

File: src/MICore/LaunchOptions.cs [hunk 2]
```diff
@@ -597,6 +597,11 @@ public LaunchCompleteCommand LaunchCompleteCommand
             }
         }
 
+        public static LaunchOptions GetInstance(HostConfigurationStore configStore, string exePath, string args, string dir, string options, IDeviceAppLauncherEventCallback eventCallback, TargetEngine targetEngine)
+        {
+            return GetInstance(configStore, exePath, args, dir, options, eventCallback, targetEngine, null);
+        }
+
```
Comment (edumunoz) [inline src/MICore/LaunchOptions.cs]: nit: remove whitespace

**Comment by faxue-msft** [comment]
@gregg-miskelly @jacdavis @rajkumar42

File: src/MICore/LaunchOptions.cs [hunk 3]
```diff
@@ -597,6 +597,11 @@ public LaunchCompleteCommand LaunchCompleteCommand
             }
         }
 
+        public static LaunchOptions GetInstance(HostConfigurationStore configStore, string exePath, string args, string dir, string options, IDeviceAppLauncherEventCallback eventCallback, TargetEngine targetEngine)
```
Comment (gregg-miskelly) [inline src/MICore/LaunchOptions.cs]: Might want to add a comment that Kofe depends on this.

**Comment by gregg-miskelly** [comment]
Otherwise looks good to me.

File: src/MICore/LaunchOptions.cs [hunk 4]
```diff
@@ -597,6 +597,12 @@ public LaunchCompleteCommand LaunchCompleteCommand
             }
         }
 
+        // Kofe debubgger depends on this method to generate LaunchOptions
```
Comment (rajkumar42) [inline src/MICore/LaunchOptions.cs]: typo: debugger :)

**Pull request merged**